aeron-driver/src/test/c/aeron_network_publication_test.cpp

```diff
@@ -189,8 +189,11 @@ class NetworkPublicationTest : public testing::Test
             aeron_counter_set_release(snd_lmt_position.value_addr, initial_position);
         }
 
+        aeron_driver_context_t *context = nullptr;
+        aeron_driver_context_init(&context);
```

Comment:
The context needs to be closed at the end of each test. Best move it to the class field and initialize in the constructor.

Alternative is to not call `aeron_driver_context_init` and only initialize necessary fields of the struct. You'll still need to free the memory though.